pages/DSNP/BatchPublications.md

```diff
@@ -71,3 +76,10 @@ Applications need to know if a given Batch file has any information they are int
 
 1. Cassandra, RocksDB, CouchDB, MongoDB, and HBASE were rejected since DSNP data needs neither a database for storage nor the overhead of one. Each of these was designed for use cases ranging from somewhat to drastically different than the DSNP network.
 1. JSON, BSON, and SQLite, while used for storage sometimes, are intended for serialization. They are schemaless, which results in redundant information and therefore a larger size than formats with schemas. They also don't support Bloom filters; instead, indexing would be required, or new batches would need to be downloaded entirely.  The exception is SQLite, which does support more advanced queries; however, it was designed for in-memory storage.
+
+### Batch Correctness and Order
+
+Batch correctness is immutable and usually in part based on the validation of the delegation of authors listed inside the batch to the publisher.
+Due to the nature of distributed systems, it is possible that a race condition occurs such that a user's delegation revocation presents before a Batch that contains a message from that user via the revoked delegate.
+While those individual messages should be considered invalid, allowing amount of time to pass before considering the entire batch invalid is suggested.
```

Comment:
We can say acceptance window ?

Reply:
I like that. I think (wrt your other comment on Identity.md) we can make it a leading window only as well! I'll try something out and let me know.